### DISCUSSION TITLE
Add bimonthly/Invoices

### DISCUSSION THREAD

**Discussion by capi1992** [firefly-iii/firefly-iii#6959] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

I would like that FireFly3 in the Invoice part that we can add bimonthly its would be great.

### Solution

Help someone that as bimonthly invoices !

**Comment by JC5** [comment, marked as answer]
That's a great idea! But you can already do that. Set "skip" to 1 to make an invoice bi-monthly.
